### ISSUE TITLE
Where to activate show/edit in rest filters

### ISSUE THREAD

**Issue by tjb1187** [airesvsg/acf-to-rest-api#254]
Hi, I'm trying to active the filters per:

https://github.com/airesvsg/acf-to-rest-api#editing-the-fields

Could you elaborate on which file to add the following code in order to active the filter options:

`// Enable the option show in rest
add_filter( 'acf/rest_api/field_settings/show_in_rest', '__return_true' );

// Enable the option edit in rest
add_filter( 'acf/rest_api/field_settings/edit_in_rest', '__return_true' );`

I know it says admin, but is there a specific file? I tried several but it did work. Thanks

**Comment by airesvsg** [comment]
Hi,

Copy and paste in your *functions.php*

https://github.com/airesvsg/acf-to-rest-api#examples

Thanks

**Comment by tjb1187** [comment]
Thanks for the quick response. I copied the following into funcitons.php:

`// Enable the option show in rest
add_filter( 'acf/rest_api/field_settings/show_in_rest', '__return_true' );

// Enable the option edit in rest
add_filter( 'acf/rest_api/field_settings/edit_in_rest', '__return_true' );`

I still don't see the filter options to show/edit field in rest. Is there additional php that needs to be added?

Thanks

**Comment by mruknowme** [comment]
I think same problem here. I included those filters, not cached. But do not see any changes.
What would be the end point to get the list of options?
So let's say for countries select field I have multiple options, how do I get the list of available options? I thought this would be a common question but apparently not, what do people usually do then?

**Comment by ninaanatchkova** [comment]
Yes, I cannot see the options to show/edit field in rest either. I copied the filters into _functions.php_.

**Comment by Yab-kami** [comment]
same problem here, plugin settings and acf options show/edit are not available.

**Comment by robtoll** [comment]
Could be php caching? I wasn't seeing any change after editing my functions.php page. Took me a while to work out that you have to restart php after each edit if you have php opcache turned on. Doh!!
